.github/workflows/main.yml

```diff
@@ -7,46 +7,83 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghidra: ["10.0.4"]
+        ghidra: ['10.1']
```

Comment:
May as well bump this up to 10.1.1 now if you can for the full log4shell mitigation (hopefully)